src/customJs/payuButtonTool/index.js

```diff
@@ -24,17 +24,26 @@ export const getPayuButtonToolConfig = () => ({
           widget: 'link_property',
           data: {
             label: intl.formatMessage({ id: 'pay_button_link' }),
-            help: `
+            help: (
               <div
-                style="font-style: italic; font-size: 13px; line-height: 1.6; color: #525252;
-                font-family: Roboto,sans-serif;">
-                ${intl.formatMessage({ id: 'payu_help' })}
-                <a target="_blank"
-                    style="color: #33AC72; text-decoration: none;"
-                    href="${intl.formatMessage({ id: 'payu_help_link' })}">
-                    ${intl.formatMessage({ id: 'help' })}
+                style={{
+                  fontStyle: 'italic',
+                  fontSize: '13px',
+                  lineHeight: '1.6',
+                  color: '#525252',
+                  fontFamily: 'Roboto,sans-serif',
+                }}
+              >
+                {intl.formatMessage({ id: 'payu_help' })}
+                <a
+                  target="_blank"
+                  style={{ color: '#33AC72', textDecoration: 'none' }}
+                  href={intl.formatMessage({ id: 'payu_help_link' })}
+                >
+                  &nbsp;{intl.formatMessage({ id: 'help' })}
```

Comment:
Is this ` &nbsp;` ok? it was not present in the previous implementation.

Reply:
This was no present because the component was treated like string, now using React the spaces in blank are ignored.

Reply:
We can use a `padding-left` but I think that is better use a the empty space of html.